yarn.lock

```diff
@@ -3689,7 +3689,7 @@ semver-diff@^2.0.0:
   version "5.3.0"
   resolved "https://registry.yarnpkg.com/semver/-/semver-5.3.0.tgz#9b2ce5d3de02d17c6012ad326aa6b4d0cf54f94f"
 
-semver@^5.5.0:
+semver@^5.4.1, semver@^5.5.0:
```

Comment:
Do you know why this yarn.lock change is here @stof? Can you drop it? Even if necessary, this does not seem relevant to this patch.

Reply:
that was done by `yarn install`. Maybe I'm using a newer version of Yarn than when this lock file was generated, with a bug fixed in it.

I inspected the lock file, and there is indeed a package depending on `semver@^5.4.1`. So this change totally make sense. I suspect a bug in an older version of Yarn making it disappear.

Reply:
I can of course drop it, but any contributor running `yarn` locally to install the dependencies will re-submit it again.

Reply:
Let me see if I can upgrade my yarn locally and repro the change separately.